person.rb

```diff
@@ -0,0 +1,22 @@
+# the class of Person
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    true if of_age? || @parent_permission
+  end
+
+  private
+
+  def of_age?
+    true if @age >= 18
```

Comment:
- [Optional] Methods that ends with an exclamation mark by default results in a boolean value of true or false. No needs to explicitly use `true` or `false` again. Hints

```
def of_age?
     @age >= 18
end
```